src/main/java/org/broadinstitute/hellbender/engine/filters/PlatformReadFilter.java

```diff
@@ -5,17 +5,24 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 import org.broadinstitute.hellbender.utils.read.ReadUtils;
 
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
 import java.util.Set;
 
 /**
  * Keep only reads that match th PL attribute.
  * Matching is done by case-insensitive substring matching
  * (checking if the read's platform tag contains the given string).
  */
-public final class PlatformReadFilter implements ReadFilter {
+public final class PlatformReadFilter implements ReadFilter, CommandLineFilter, Serializable {
     private static final long serialVersionUID = 1L;
-    @Argument(fullName = "PLFilterName", shortName = "PLFilterName", doc="Keep reads with RG:PL attribute containing this string", optional=true)
-    public Set<String> PLFilterNames;
+
+    private final static String platformArgName = "PLFilterName";
+
+    @Argument(fullName = platformArgName, shortName = "PLFilterName", doc="Keep reads with RG:PL attribute containing this string", optional=true)
```

Comment:
It may be possible to leverage jotp's availableIf to do this more easily, see also requiredIf and requiredUnless http://pholser.github.io/jopt-simple/examples.html